moit-api/src/main/kotlin/com/mashup/moit/fine/controller/dto/FineConfirmRequest.kt

```diff
@@ -0,0 +1,12 @@
+package com.mashup.moit.fine.controller.dto
+
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.NotBlank
+
+
+@Schema(description = "벌금 송금 평가 RequestBody")
+data class FineEvaluateRequest(
+    @Schema(description = "벌금 송금 승인 여부")
+    @field:NotBlank
```

Comment:
```suggestion
    @field:NotNull
```
boolean 타입이라 NotNull 로 바꾸면 좋을 것 같습니다!

Reply:
오우 놓쳤네요. 수정했습니다.